praktikumsplaner-frontend/frontend/src/components/TheErrorDialog.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+    <v-dialog
+        id="error"
+        v-model="show"
+        persistent
+        width="800"
+    >
+        <v-card>
+            <v-card-title>
+                {{ title }}
+            </v-card-title>
+            <v-card-text>
+                <p class="message">
+                    {{ message }}
+                </p>
+            </v-card-text>
+            <v-card-actions>
+                <v-spacer />
+                <v-btn
+                    color="primary"
+                    @click="close"
+                >
+                    Schließen
+                </v-btn>
+            </v-card-actions>
+        </v-card>
+    </v-dialog>
+</template>
+
+<script setup lang="ts">
+import { ref, watch } from "vue";
+import { useErrorStore } from "@/stores/error";
+
+const show = ref(false);
+const message = ref("");
+const title = ref("");
+
+const errorStore = useErrorStore();
+
+watch(
+    () => errorStore.title,
+    () => (title.value = errorStore.title ?? "")
+);
+
+watch(
+    () => errorStore.message,
+    () => (message.value = errorStore.message ?? "")
+);
+
+watch(
```

Comment:
when connecting the two, calling the dialog will only be possible once. This is because the store value will stay true while the show attribute will go back to being false, and calling a new UserErrorDialog will not change the already true value, so it doesn't trigger the watcher

Reply:
Does this also apply when changing the store value via a action? So the moment the user desires to close the dialog an action on the store is called to set the show state of the store to false.